### DISCUSSION TITLE
Put table in disabled mode when editing a row

### DISCUSSION THREAD

**Discussion by groyee** [KevinVandy/material-react-table#285] [Ideas]
I think it would be really cool if the table will be disabled during a row edit except the editing row. Something like that:

<img width="1710" alt="Screen Shot 2565-12-27 at 02 07 07" src="https://user-images.githubusercontent.com/194016/209577390-ed50e7d8-53b2-4e6b-a3a5-a4b16582f1d4.png">

**Comment by KevinVandy** [comment]
You can do something like this

```jsx
<MaterialReactTable
  muiTableBodyRowProps={({ row, table }) => ({
    hover: !table.getState().editingRow,
    sx: {
      opacity: !table.getState().editingRow || table.getState().editingRow.id === row.id ? 1 : 0.5
    }
  })}
/>
```

**Comment by groyee** [comment]
Yep... you right. I did something similar here with pointerEvents as well... not sure if it should be a feature in the core table or not...

https://github.com/KevinVandy/material-react-table/pull/286

**Reply by AdityaBhadange** [reply]
`opacity:
            // @ts-ignore
            !table?.getState()?.editingRow || table?.getState()?.editingRow?.id === row?.id
              ? 1
              : 0.3,
pointerEvents:
            // @ts-ignore
            !table?.getState()?.editingRow || table?.getState()?.editingRow?.id === row?.id
              ? 'auto'
              : 'none'`

Use this to blur the other rows and also disable mouse click.
